jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,654 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_params = {\n",
+    "    \"name\": \"basemap order with geometry\",\n",
+    "    \"source_type\": \"basemaps\",\n",
+    "    \"order_type\": \"partial\",\n",
+    "    \"products\": [\n",
+    "        {\n",
+    "            \"mosaic_name\": \"global_monthly_2022_01_mosaic\",\n",
+    "            \"geometry\": {\n",
+    "                \"type\": \"Polygon\",\n",
+    "                \"coordinates\": [\n",
+    "                   [\n",
+    "                       [4.607406, 52.353994],\n",
+    "                       [4.680005, 52.353994],\n",
+    "                       [4.680005, 52.395523],\n",
+    "                       [4.607406, 52.395523],\n",
+    "                       [4.607406, 52.353994]\n",
+    "                   ]\n",
+    "                ]\n",
+    "            }\n",
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\"merge\": {}},\n",
+    "        {\"clip\": {}}\n",
+    "    ],\n",
+    "    \"delivery\": {\n",
+    "        \"google_cloud_storage\": {\n",
+    "            \"bucket\": \"devrel-notebooks\",\n",
+    "            \"credentials\": GCP_CREDENTIALS,\n",
+    "            \"path_prefix\": \"basemaps-to-cloud/\"\n",
+    "        }\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Print the response object\n",
+    "---\n",
+    "Retrieve the response object as text to print out to check if there are errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"_links\":{\"_self\":\"https://api.planet.com/compute/ops/orders/v2/af23033f-bc81-4515-a41f-7ad78ce63ddf\"},\"created_on\":\"2023-04-04T21:05:08.898Z\",\"delivery\":{\"google_cloud_storage\":{\"bucket\":\"devrel-notebooks\",\"credentials\":\"<REDACTED>\",\"path_prefix\":\"basemaps-to-cloud/\"}},\"error_hints\":[],\"id\":\"af23033f-bc81-4515-a41f-7ad78ce63ddf\",\"last_message\":\"Preparing order\",\"last_modified\":\"2023-04-04T21:05:08.898Z\",\"name\":\"basemap order with geometry\",\"order_type\":\"partial\",\"products\":[{\"geometry\":{\"coordinates\":[[[4.607406,52.353994],[4.680005,52.353994],[4.680005,52.395523],[4.607406,52.395523],[4.607406,52.353994]]],\"type\":\"Polygon\"},\"mosaic_name\":\"global_monthly_2022_01_mosaic\"}],\"source_type\":\"basemaps\",\"state\":\"queued\",\"tools\":[{\"merge\":{}},{\"clip\":{}}]}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "ORDERS_API_URL = 'https://api.planet.com/compute/ops/orders/v2'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "param_req = requests.post(ORDERS_API_URL, data=json.dumps(\n",
```

Comment:
Same comments here as above - you should be using `session.post`

Also, instead of `data=json.dumps(order_params)`, you can just pass `json=order_params`

So, with both of those:
```
param_req = session.post(ORDERS_API_URL, json=order_params)
```